RetailCoder.VBE/Extension.cs

```diff
@@ -221,7 +221,9 @@ private void Startup()
             catch (Exception e)
             {
                 _logger.Log(LogLevel.Fatal, e, "Startup sequence threw an unexpected exception.");
-                //throw; // <<~ uncomment to crash the process
+#if DEBUG
+                throw; // <<~ uncomment to crash the process
```

Comment:
IMO, I'd remove the comment altogether. Or put in the "start a WW III" comment instead.

Reply:
That's the throw discussed in #3563